snapshot/lib/snapshot/runner.rb

```diff
@@ -204,6 +207,21 @@ def erase_simulator(device_type)
       `xcrun simctl erase #{device_udid} &> /dev/null`
     end
 
+    def localize_simulator(device_type, language, locale)
+      apple_locale = language.sub("-", "_")
```

Comment:
So this is interesting. Snapshot takes a *Language Identifier* and formally there are only three forms to specify those:

* Language (nl)
* Language-Region (en-CA)
* Language-Script (zh-Hans, zh-Hant)

These are often confused with *Locale Identifiers* where the triplet that you mention does exist:

* Language (nl)
* Language_Region (en_GB)
* Language-Script (az-Arab)
* Language-Script_Region (zh-Hans_HK)

(Note the interesting mix of underscore and hypen)

So, it's complicated.

Assuming Snapshot takes a *Language Identifier* in its config, I need to figure out if the mapping I do now to *Apple _Locale_* and *Apple _Languages_* is actually correct.

(I think iOS is pretty lenient in what it accepts there and may make the best of the situation, but it is entirely possible that I do not pass it the right things)

Needs a little more research.